src/components/Table/Table.js

```diff
@@ -42,6 +42,35 @@ class Table extends Component {
 
 
     }
+    //полная дичь. я почти все это время разбиралась в модулях)))
+
+    removeRowFromTable = (value) => {
+        Array.prototype.remove = function() {
```

Comment:
do the same logic for removing items but without prototypes, create your own custom method for this particular case.